var/spack/repos/builtin/packages/octave/package.py

```diff
@@ -164,6 +165,20 @@ def configure_args(self):
             "--with-lapack=%s" % spec['lapack'].libs.ld_flags
         ])
 
+        if '^mkl' in spec and 'gfortran' in self.compiler.fc:
```

Comment:
This is still separate from the above block, wouldn't it be better to combine into one block that sets the blas and lapack options at once?

Reply:
Yes, thanks for noticing it. I am following https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/r/package.py and they have it that way too. I'll make the change in a moment.

Reply:
Just pushed the update with the combined block, sorry for the delay, got sidetracked.

Reply:
Excellent :+1: